apps/console/project.json

```diff
@@ -121,7 +121,7 @@
             "options": {
                 "commands": [
                     "pnpm nx run console:build-prepare",
-                    "pnpm nx run console:build-base",
+                    "pnpm nx run console:build-base --verbose",
```

Comment:
Let's add this under `configurations[ENV]`.
Also, we use NX plugins in other modules like `react-components`. Check if we need to add this there as well.
```js
"configurations": {
    "development": {
        ...,
        "vendorChunk": true,
        "verbose": true
    },
    "production": {
        ...,
        "vendorChunk": true,
        "verbose": true
    }
}
```

Reply:
Addressed with 7c7393c, 50898af, and 3d56097